### PR TITLE
Building kedge & winch without CGO.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN cd ${GOPATH}/src/github.com/improbable-eng/kedge && dep ensure
 
 ARG BUILD_VERSION
 RUN echo "Installing Kedge with version ${BUILD_VERSION}"
-RUN go install -ldflags "-X main.BuildVersion=${BUILD_VERSION}" github.com/improbable-eng/kedge/cmd/kedge
-RUN go install -ldflags "-X main.BuildVersion=${BUILD_VERSION}" github.com/improbable-eng/kedge/cmd/winch
+RUN CGO_ENABLED=0 go install -ldflags "-X main.BuildVersion=${BUILD_VERSION}" github.com/improbable-eng/kedge/cmd/kedge
```
Comment (stefan-improbable) [inline Dockerfile]: Nit: can you add a comment here on why is this necessary?

**Comment by bwplotka** [comment]
This is for debug purposes, and it does not improve anything - but get some overview of what is happening (:
